### PR TITLE
Tickets/dm 41806

### PR DESCRIPTION
**Pull request by dsanmartim**
Hi Tiago,

I have the unit test passing, but I realized that some auxiliary methods added to the mock modules may be unnecessary. Waiting for your feedback to merge it and finish the corresponding SAL scripts.

### CODE REVIEW

File: python/lsst/ts/observatory/control/auxtel/atcs.py [hunk 1]
```diff
@@ -961,6 +962,139 @@ async def close_dome(self, force: bool = True) -> None:
...
+        """Close ATDome dropout door"""
+
+        dropout_door_state = await self.rem.atdome.evt_dropoutDoorState.aget(
+            timeout=self.fast_timeout
+        )
+        if dropout_door_state.state == ATDome.ShutterDoorState.OPENED:
+            self.log.debug("Closing dome dropout door...")
+            await self.rem.atdome.cmd_moveShutterDropoutDoor.set_start(
+                open=False, timeout=self.open_dropout_door_time
+            )
```
Comment (tribeiro) [inline python/lsst/ts/observatory/control/auxtel/atcs.py:1022]: Need to wait for the dropout to close right?

File: python/lsst/ts/observatory/control/mock/atcs_mock.py
Comment (tribeiro) [inline python/lsst/ts/observatory/control/mock/atcs_mock.py:1]: Nice of you to update this, but this needs to be deprecated.

**Review by tribeiro — COMMENTED** [review]
Looks good! I only have a couple comments I hope you consider before I can approve it. 


Also, can you please fix the GitHub lint action? you need to update [this](https://github.com/lsst-ts/ts_observatory_control/blob/412b02b83039acdcbde64d3028ecd9f98e42ae2f/.github/workflows/lint.yaml#L20) with the following:

```
$CONDA/bin/conda install -c lsstts -c conda-forge pre-commit ts-pre-commit-config python=3.11 -y
```

File: python/lsst/ts/observatory/control/auxtel/atcs.py (hunk 1 above)
Comment (dsanmartim) [reply python/lsst/ts/observatory/control/auxtel/atcs.py:1028]: Alright. Take a look to see if it is ok now.

**Review by tribeiro — APPROVED** [review]
looks good! thanks!

**Pull request merged**